File: py/Descuento.py
valor = int (input("ingrese el valor de su compra: "))
descuento = valor * 0.10
total = valor - descuento

if valor >= 100000:
    print ("descuento de:", descuento)
    print ("Total:", total, "pesos")
else:
    print ("no aplica descuento")
    print ("total:", valor, "pesos")

File: Descuento.js
let valor = 350000
let descuento = valor * 0.10
let total = valor - descuento

if (valor >= 100000) {
    console.log ("descuento:", descuento)
    console.log ("total:", total, "pesos");
}
else {
    console.log ("no aplica descuento")
    console.log ("total:", valor, "pesos")
}